### PR TITLE
Documentation for FA processor

### PR DESCRIPTION
**Pull request by rtso**

### CODE REVIEW

**Comment by rtso** [comment]
* **#546** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/546?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rtso and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs [hunk 1]
```diff
@@ -394,6 +394,7 @@ impl FungibleAssetBalance {
                     last_transaction_timestamp: txn_timestamp,
                     token_standard: TokenStandard::V1.to_string(),
                 };
+                // What is this for?
```
Comment (bowenyang007) [inline rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs:397]: V1 events don't have asset type. It's just "0x1::coin::WithdrawEvent" (or something like that) w/ the address and amount of APT.

**Review by bowenyang007 — APPROVED** [review]
thanks for adding the comments, should we also summarize this in a doc somewhere? We could also add this to dev docs post migration.

File: rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs (hunk 1 above)
Comment (rtso) [reply rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs:397]: Ok I see you've commented about it in the main file so I'll leave this.

**Comment by rtso** [comment]
Dev docs post migration makes sense to me.

**Comment by rtso** [comment]
### Merge activity

* **Oct 23, 4:21 PM EDT**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/546).

**Pull request merged**